cluster/ubuntu-cluster/configure.sh

```diff
@@ -21,6 +21,10 @@
 
 set -e
 
+CONFIG_DIR=`dirname "$0"`
+CONFIG_DIR=`cd "$CONFIG_DIR"; pwd`
```

Comment:
Line 25 will change the path to a full path.
Indeed it makes no difference here between full path and relative path so far. You can disgard that line when merging.
I just habitually do that in my own scripts.